src/components/Badge/Badge.story.tsx

```diff
@@ -1,29 +1,32 @@
 import * as React from 'react';
 import { Story } from '@storybook/react';
-import Badge from './Badge';
 import MailIcon from '@material-ui/icons/Mail';
-import { BadgeProps } from './Badge.types';
+import Badge from '.';
+import { BadgeProps } from '.';
 
 
 export default {
     component: Badge,
     title: 'Data Display/Badge',
 };
 
-
-export const Sandbox: Story<BadgeProps> = (args) => <Badge {...args}><MailIcon /></Badge>;
-Sandbox.args = {
-    color: 'primary',
+const defaultArgs = {
     invisible: false,
-    variant: 'standard',
-    overlap: 'rectangle',
-    badgeContent: '4',
     showZero: false,
     max: 99,
+    badgeContent: '4',
     anchorOrigin: {
-        vertical: 'top',
-        horizontal: 'right',
+        vertical: 'top' as "top",
+        horizontal: 'right' as "right",
     },
+    overlap: 'rectangle' as "rectangle",
+    variant: 'standard' as "standard",
```

Comment:
Do these `as` calls serve a specific purpose? I don't recall seeing these used in other sandboxes...